src/EnergyPlus/DataAirflowNetwork.hh

```diff
@@ -1021,6 +1031,110 @@ namespace DataAirflowNetwork {
 
 	};
 
+	struct IntraZoneNodeProp // Intra zone node data
+	{
+		// Members
+		std::string Name; // Name of node
+		std::string RAFNNodeName; // RoomAir model node name
+		Real64 Height; // Nodal height
+		int RAFNNodeNum; // RoomAir model node number
+		int ZoneNum; // Zone number
+		int AFNZoneNum; // MultiZone number
+
+		// Default Constructor
+		IntraZoneNodeProp( ) :
+			Height( 0.0 ),
+			RAFNNodeNum( 0 ),
+			ZoneNum( 0 ),
+			AFNZoneNum( 0 )
+		{}
+
+		// Member Constructor
+		IntraZoneNodeProp(
+			std::string const & Name, // Name of node
+			std::string const & RAFNNodeName, // RoomAir model node name
+			Real64 const Height, // Nodal height
+			int const RAFNNodeNum, // RoomAir model node number
+			int const ZoneNum // Zone number
+			) :
+			Name( Name ),
+			RAFNNodeName( RAFNNodeName ),
+			Height( Height ),
+			RAFNNodeNum( RAFNNodeNum ),
+			ZoneNum( ZoneNum ),
+			AFNZoneNum( AFNZoneNum )
```

Comment:
@lgu1234 Is there an argument missing here? This initializes AFNZoneNum with itself.